src/Atom.res

```diff
@@ -38,7 +38,7 @@ type getter
 let value = getter->Jotai.Atom.get(atom)
 ```
 ")
-let get = (type value, get: getter, atom: t<value, _, [> Permissions.r]>): value =>
+let get = (type value, get: getter, atom: t<value, 'value, [> Permissions.r]>): value =>
```

Comment:
Functionally identical I use `'value` instead of `_` for consistency only, thank you @eccentric-j for pointing this out 👍